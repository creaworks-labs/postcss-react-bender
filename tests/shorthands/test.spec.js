var { processFileAndMatchSnapshot } = require("../utils");

it.skip("process basic definitions with shorthanded props", () => {
  return processFileAndMatchSnapshot("./tests/shorthands/fixtures/basic.scss");
});

it("process pure style definition with shorthanded props", () => {
  return processFileAndMatchSnapshot("./tests/shorthands/fixtures/pure.scss");
});
