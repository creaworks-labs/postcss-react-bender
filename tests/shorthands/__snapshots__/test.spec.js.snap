// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`process pure style definition with shorthanded props 1`] = `
Object {
  "circle": Object {
    "alignSelf": "center",
    "borderBottomWidth": 1,
    "borderLeftWidth": 50,
    "borderRightWidth": 50,
    "borderTopWidth": 50,
    "fontFamily": "Georgia, serif",
    "fontSize": 12,
    "fontStyle": "italic",
    "fontVariant": Array [
      "small-caps",
    ],
    "fontWeight": "500",
    "lineHeight": 30,
  },
}
`;
